css-two-column-layout/styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+  font-family: 'Karla', sans-serif;
+}
+
+button {
+  border: 2px solid #d14c45;
+  background-color: white;
+  color: #d14c45;
+  padding: 14px 48px;
+  font-size: 16px;
+  cursor: pointer;
+  font-weight: bold;
+}
+
+h1 {
+  text-align: center;
+  font-size: 3em;
+  font-stretch: semi-expanded;
+  letter-spacing: 2px;
+  font-weight: bold;
+}
+
+img {
+  max-width: 120px;
+  padding-bottom: 50px;
+  margin-bottom: 50px;
+  position: absolute;
+}
```

Comment:
Why are all `img` elements absolute positioned? That is a massive change, and you don't seem to be using any box offset properties, so I don't see the purpose either 🤔